tests/libvirt/models/compute/server_tests.rb

```diff
@@ -1,10 +1,50 @@
+RealFile = File
+class FakeFile < RealFile
+  def self.file?(path)
+    if path == '/etc/foreman/ceph.conf'
+      return true
+    else
+      return RealFile.file(path)
+    end
```

Comment:
To remove some RuboCop concerns, perhaps reduce it to
```suggestion
    path == '/etc/foreman/ceph.conf' || RealFile.file(path)
```

Reply:
Nice ! I will change that. I'm more used to developing in Python than in Ruby.